reads_filtration/module_wraper.py

```diff
@@ -90,6 +89,15 @@ def run_first_phase(fastq_path, first_phase_output_path, logs_dir, barcode2sampl
     else:
         logger.info(f'{datetime.datetime.now()}: skipping count_and_collapse_duplicates.py ({done_path} exists)')
 
+
+    if stop_machines:
+        name_script = 'tools/stop_machine_aws.py'
+        done_path = f'{logs_dir}/stop_machines_done.txt'
+        module_parameters = [done_path,'--type_machines', type_machines_to_stop, '-v' if verbose else '']
+        cmd = fetch_cmd(name_script, module_parameters,verbose, error_path, done_path )
+        wait_for_results('stop_machine_aws', logs_dir, num_of_expected_results=1, example_cmd=cmd,
+                         error_file_path=error_path, suffix='stop_machines_done.txt')
```

Comment:
Instead of cloning the code we can put it in a shared file.
Also, not sure why we need to run the script asynchronously